### DISCUSSION TITLE
Why don't you call this library AwesomePDF?

### DISCUSSION THREAD

**Discussion by adurmus** [QuestPDF/QuestPDF#311] [Polls]
Finally an Open Source Pdf library that does all PDF operations. The usage is simple. and it supports the latest .net. It is fun to generate PDFs now in .NET.  That's why I would call this library as **AwesomePDF** :) And tell me about the preview. love it!. Great job!

**Comment by iivanmendozaa** [comment]
Amazing library, I recently implement it on Blazor WebAssembly app.

**Comment by Reid-Anders** [comment]
I also came here to compliment the library. I think it's simply awesome! Makes my life so much easier

**Comment by DawidRobertNowak** [comment]
Thank you for the kind words — we're glad you enjoy using QuestPDF! We'll stick with the name for now, but the appreciation means a lot. 😄
